source/integrations/zoom.rst

```diff
@@ -25,11 +25,15 @@ Zoom Setup Guide
 .. image:: ../images/zoom_system_console.png
 
 2. If you're using a self-hosted private cloud or on-premise Zoom server, enter the **Zoom URL** and **Zoom API URL** for the Zoom server, for example ``https://yourzoom.com`` and ``https://api.yourzoom.com/v2`` respectively. Leave blank if you're using Zoom's vendor-hosted SaaS service.
+
 3. Set the **API Key** and **API Secret**, generated by Zoom and used to create meetings and pull user data:
 
-  - Go to https://developer.zoom.us/me/ and log in.
-  - Switch to the **API** tab and click **Enable**, if not already enabled.
-  - Switch back to the **API** tab and copy the **API Key** and **API Secret**.
+  - Go to https://marketplace.zoom.us/ and log in.
+  - In the top left click on **Develop** and then **Build App**
+  - Enter a name for your app and select **Account-level App**, also disable **Intend to publish**.
+  - Select **JWT API Credentials** as authentication type.
+  - Enter the needed credentials for your app.
```

Comment:
Are these Zoom credentials?

Reply:
Good point this could be confused with the app credential tab. Will reword. It's actually the Information tab where you need to input your name, email and organization.  
Those are mandatory fields so you can't continue without inputting them.